README.md

```diff
@@ -11,19 +11,80 @@ Get started with the sample API scripts with zero configuration. Click on the
 "Gitpod ready-to-code" button above.
 
 # Getting started for Unix based systems (Mac OS & Linux)
-
+1. Clone this repo onto your local machine so you have access to all the files and can change them as you need.
+    ```bash
+    git clone git@github.com:vegaprotocol/sample-api-scripts.git
+    ```
+    This should create you a folder named `sample-api-scripts` that you will use for the rest of this README.
 1. Copy or rename the `credentials-template` file as `credentials`.  
-*Note: When running on Gitpod this is already done for you.*
+    *Note: When running on Gitpod this is already done for you.*
+    ```bash
+    cp credentials-template credentials
+    ```
 1. Edit the `credentials` file. (`nano` and `vim` are installed, or use the built-in Gitpod text editor.)
+    ```bash
+    nano credentials
+    ```
```

Comment:
Considering the entry-level of this documentation, and since we advise to use `nano`, we shall indicate the way to save the modification and the way to exit it, in my opinion.